.env.docker

```diff
@@ -0,0 +1,15 @@
+MONGO_INITDB_ROOT_USERNAME=user
+MONGO_INITDB_ROOT_PASSWORD=pass
+
+HOST=localhost
+PROTOCOL=http
+PORT=5000
+MONGODB=mongodb://user:pass@cla-mongo/admin
+
+GITHUB_CLIENT=xxxxx
+GITHUB_SECRET=xxxxx
+GITHUB_TOKEN=xxxxx
+GITHUB_ADMIN_USERS=ghusername
+
+HTTP_PROXY=http://localhost:3128
+HTTPS_PROXY=http://localhost:3128
```

Comment:
Shouldn't we use only `HTTP_PROXY`? I'm assuming we should in the next comments. If not, ignore those.